cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -18,6 +18,51 @@ spec:
     - rcc
   scope: Namespaced
   version: v1beta1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            cephVersion:
+              properties:
+                allowUnsupported:
+                  type: boolean
+                image:
+                  type: string
+                name:
+                  pattern: ^(luminous|mimic|nautilus)$
```

Comment:
I'm just sort of rethinking this a little. I think there could be some benefit to spinning up a ceph container before the operator starts operations with the `ceph --version` command to detect the version automatically. I think this removes the potential for user error if they fail to specify `name` or fail to update it when they should. What are your thoughts?